docs/addons/karpenter.md

```diff
@@ -38,37 +40,32 @@ const karpenterAddOn = new blueprints.addons.KarpenterAddOn({
       annotations: {
           "eks-blueprints/owner": "young"
       },
-      taints: [{
-          key: "workload",
-          value: "test",
-          effect: "NoSchedule",
-      }],
       requirements: [
-          { key: 'node.kubernetes.io/instance-type', operator: 'In', values: ['m5.2xlarge'] },
-          { key: 'topology.kubernetes.io/zone', operator: 'In', values: ['us-west-2a','us-west-2b', 'us-west-2c']},
+          { key: 'node.kubernetes.io/instance-type', operator: 'In', values: ['m5.large'] },
+          { key: 'topology.kubernetes.io/zone', operator: 'In', values: [`${region}a`, `${region}b`, `${region}c`] },
```

Comment:
Should these be having a dash before the zone: `${region}-a` as opposed to `${region}a`?

Reply:
No dash should have it resolve to us-west-2a, us-west-2b, and us-west-2c, which matches the example

Reply:
nm, not sure what I am seeing in the morning.